catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -207,6 +207,11 @@
         <bundle>mvn:ddf.catalog.security/catalog-security-filter/${project.version}</bundle>
     </feature>
 
+    <feature name="catalog-security-operationplugin" install="manual" version="${project.version}"
```

Comment:
Should this feature have been rather named "catalog-security-operation-plugin"?
Also file could use a reformat (separate commit?)